src/client/AM.PA.MonitoringTool/SlackTracker/Settings.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Created by Rohit Kaushik (f20150115@goa.bits-pilani.ac.in) at the University of Zurich
+// Created: 2018-07-02
+// 
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SlackTracker
+{
+    internal class Settings
+    {
+        // Daemon
+        internal const string TRACKER_NAME = "Slack Tracker";
+        internal const string TRACKER_ENABLED_SETTING = "SlackTrackerEnabled";
+        internal const int SYNCHRONIZE_INTERVAL = 8 * 20 * 1000; //1 minutes
+
+        //Database Tables names
+        internal static readonly string DOWNLOAD_TABLE_NAME = "slack_downloads";
+        internal const string CHANNELS_TABLE_NAME = "SlackChannels";
+        internal const string LOG_TABLE_NAME = "SlackLogs";
+        internal const string USER_MENTION_TABLE_NAME = "MessageDirectedTo";
```

Comment:
please rename to "SlackMessageDirectedTo". This will group all Slack-Tracker tables together and make it easier for us and the user to spot which data goes together.